test/pageobjects/PublicationsPage.js

```diff
@@ -2,20 +2,29 @@ const Page = require("./Page");
 
 class PublicationsPage extends Page{
     get h6(){
-        return $('h6');
+        return $('//h6');
     }
-    get textAddPublication(){
+
+    get buttonAddPublication(){
```

Comment:
```suggestion
    get addPublicationButton(){
```

consistency - saveButton, AddPublicationPage, publicationList